crates/rpc/src/rpc.rs

```diff
@@ -52,10 +54,17 @@ impl Server {
 }
 
 impl Rpc for Server {
+    fn get_featured_listings(&self) -> Result<Vec<Listing>> {
+        let db = self.db()?;
+
+        listings_triple_join::load(|q| q.limit(4), &db, Local::now().naive_utc())
+            .map_err(internal_error("Failed to load listings"))
+    }
+
     fn get_listings(&self) -> Result<Vec<Listing>> {
         let db = self.db()?;
 
-        listings_triple_join::load(|q| q, &db, Local::now().naive_utc())
+        listings_triple_join::load(|q| q.offset(4), &db, Local::now().naive_utc())
             .map_err(internal_error("Failed to load listings"))
     }
```

Comment:
I think it might be good to rename this method to something like `getNonFeaturedListings` for clarity.  Since the original `getListings` was kind of a stopgap, I'd be fine with removing it altogether if we're no longer using it.